package.json

```diff
@@ -39,7 +39,7 @@
     "express": "^4.13.3",
     "fetch-mock": "^3.0.0",
     "lintspaces-cli": "^0.1.1",
-    "mocha": "^2.2.4",
+    "mocha": "^5.0.0",
```

Comment:
Do you now need to add `--exit` after the `mocha` command?

Reply:
--exit                                  force shutdown of the event loop after test run: mocha will call process.exit

Reply:
Normally you shouldn't use --exit as far as I understand.